### PR TITLE
feat: add inspector method plugin + fix theme

### LINKED ISSUE #253: Font color in darkmode
When using the dark mode on latest Chrome or Firefox on Ubuntu 18.04, the descriptions font color is black as well.

Image with out highlighting. 
image

Image with highlighting the text to see it.
image

### PR DESCRIPTION
**Pull request by shanejonas**
fixes #253 
inspector_playground_1

### CODE REVIEW

File: src/plugins/InspectorPlugin.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  let exampleParams;
+  if (method && method.examples && method.examples[examplePosition]) {
+    example = method.examples[examplePosition] as ExamplePairingObject;
+    exampleParams = (example.params as ExampleObject[]).map((p) => p.value);
+  }
+  return (
+    <Grid style={{ height: "300px", width: "100%", overflowY: "auto" }}>
+      <Inspector
+        request={{ method: method.name, params: exampleParams || [] }}
+        url={searchUrl && searchUrl.includes(".json") ? null : searchUrl}
```
Comment (zcstarr) [inline src/plugins/InspectorPlugin.tsx]: is this the case for rpc-discover or is this not affected by that?

Comment (zcstarr) [reply src/plugins/InspectorPlugin.tsx]: I guess should the endpoint support the discover endpoint optionally or is this just better to limit the scopefor now to be .json specific endpoints

Comment (shanejonas) [reply src/plugins/InspectorPlugin.tsx]: yea basically if it ends in `.json` i dont want to set it as a `url` for the nested `inspector`s

Comment (shanejonas) [reply src/plugins/InspectorPlugin.tsx]: It's the other way around. this sets `url: null` if searchUrl includes `.json`.

**Pull request merged**